src/pgrn-query-expand.c

```diff
@@ -502,7 +503,14 @@ pgroonga_query_expand(PG_FUNCTION_ARGS)
 	}
 	tableOID = DatumGetObjectId(tableOIDDatum);
 	currentData.table = RelationIdGetRelation(tableOID);
-
+	if (!PGRN_RELKIND_HAS_TABLE_AM(RelationGetForm(currentData.table)->relkind))
+	{
+		PGrnCheckRC(GRN_INVALID_ARGUMENT,
+					"%s the specified table isn't table: <%s>, <%s>",
```

Comment:
You should not use `%s` for `char`:

https://github.com/pgroonga/pgroonga/actions/runs/6891003014/job/18745160951?pr=375#step:6:55

```text
src/pgrn-query-expand.c:509:86: warning: format ‘%s’ expects argument of type ‘char *’, but argument 5 has type ‘int’ [-Wformat=]
  509 |                                         "%s the specified table isn't table: <%s>, <%s>",
      |                                                                                     ~^
      |                                                                                      |
      |                                                                                      char *
      |                                                                                     %d
```

(I don't think that outputting `relkind` as-is is easy to debug because we can't understand `i` means `RELKIND_INDEX` easily...)